src/frontend/scripts/cookie-panel.es6

```diff
@@ -36,25 +36,6 @@
     document.cookie = `${name}=${value || ""}; Expires=${date.toUTCString()}; Max-Age=${365 * 24 * 60 * 60}; Path=/`;
   };
 
-  // -- Temporary fix for broken cookie handling in XP
```

Comment:
Previously it was adding cookies via the back-end, and I believe something about these cookies wasn't ideal. Maybe the path or cookie name was different, so the fix was to delete and recreate them in the front-end.

I didn't look into it any further because it's not really relevant anymore. We're only creating cookies from the front-end now. 😃